frontend/src/scenes/web-analytics/tiles/WebAnalyticsTile.tsx

```diff
@@ -509,6 +514,37 @@ export const WebGoalsTile = ({
         </div>
     )
 }
+
+export const WebExternalClicksTile = ({
+    query,
+    insightProps,
+}: {
+    query: DataTableNode
+    insightProps: InsightLogicProps
+}): JSX.Element | null => {
+    const { shouldStripQueryParams } = useValues(webAnalyticsLogic)
+    const { setShouldStripQueryParams } = useActions(webAnalyticsLogic)
+    return (
+        <div className="border rounded bg-bg-light flex-1 flex flex-col">
+            <div className="flex flex-row items-center justify-end m-2 mr-4">
+                <div className="flex flex-row items-center space-x-2">
+                    <LemonSwitch
+                        label={
+                            <div className="flex flex-row space-x-2">
+                                <span>Strip query parameters</span>
+                            </div>
+                        }
+                        checked={!!shouldStripQueryParams}
```

Comment:
Why can't this just be the boolean value? Doesn't look like it's ever typed as `null`